docs/book/SUMMARY.md

```diff
@@ -1,3 +1,14 @@
+<!-- START doctoc generated TOC please keep comment here to allow auto update -->
```

Comment:
This file is a Table of Contents already...

Reply:
This would be update the next time to run doctoc with new content. It is fine.
And it has not the table already. I have inspected the source code.